### DISCUSSION TITLE
Sobre la tercera parte: Identificación de Regiones

### DISCUSSION THREAD

**Discussion by Fernanda-Bley** [IIC2133-PUC/2024-1#175] [Tarea 3] [answered]
Estaba pensando en la resolución de la tercera parte visualice el input y output. 
Hay 10 nodos con  6 aristas. Hice un dibujo de como se vería. 
![Screenshot_20240604_130939_Samsung Notes](https://github.com/IIC2133-PUC/2024-1/assets/89091540/3233eff5-b1ff-4485-ad7a-c82dfbbb56a7)
Lo que no comprendo es porque dice que hay 4 grupos no conectados, cuando el único nodo sin uniones es el 8. 
¿A que se refiere por grupos no conectados?

**Comment by 4n1t4** [comment, marked as answer]
Piénsalo como cuantos grafos totalmente conectados hay.  De forma más formal debes indicar cuantos grafos cumplen con que para cada par de vertices (a,b) existe al menos un camino factible que conecte "a" con "b". En otras palabras cuantos grafos conexos hay, lo cual en el ejemplo son 4 formados por :{8}, {5,7}, {6,9}, {0,1,2,3,4}
